scripts/modules/controllers/AddBookFormController.js

```diff
@@ -17,5 +17,5 @@ export default class AddBookFormController {
       });
       this.addBookFormViewManager.clearInputs();
     });
-  }
+  };
```

Comment:
- [ ] (Optional) It would be nice to show a message that alerts users that a book has been added after submitting inputs. In the message, you can also direct the user to click the _**List**_ link in the navbar to see the book added. 👍